duet/views/app_store_territory_source_type_report.view.lkml

```diff
@@ -0,0 +1,139 @@
+view: app_store_territory_source_type_report {
+  derived_table: {
+    sql:
+      SELECT
+        date,
+        territory,
+        source_type,
+        active_devices,
+        active_devices_last_30_days,
+        deletions,
+        installations,
+        sessions,
+        first_time_downloads,
+        redownloads,
+        total_downloads,
+        impressions,
+        impressions_unique_device,
+        page_views,
+        page_views_unique_device
+      FROM
+        mozdata.app_store.firefox_app_store_territory_source_type_report
+      FULL OUTER JOIN
+        mozdata.app_store.firefox_downloads_territory_source_type_report
+        USING (date, territory, source_type)
+      FULL OUTER JOIN
+        mozdata.app_store.firefox_usage_territory_source_type_report
+        USING (date, territory, source_type) ;;
```

Comment:
There's no aggregations in the above list so this seems like it could be another derived table or a join between views in the explore no?

Reply:
I don't disagree, but I wanted it this way so the explore is more intuitive (rather than having a bunch of separate views)